ccd-definition/CaseEvent/StateChange.json

```diff
@@ -41,7 +41,7 @@
     "Name": "Populate standard directions",
     "Description": "Populate standard directions",
     "DisplayOrder": 14,
-    "PreConditionState(s)": "Submitted",
+    "PreConditionState(s)": "*",
```

Comment:
Are we sure that we want this to end in the gatekeeping state now?

Reply:
Yes, we currently use this event when sending case to gatekeeper (which is submitted->gatekeeping state transition) and now it'll also be triggered when adding hearing details in gatekeeping state (with no state change afterwards)